lua/nvim-tree/actions/fs/create-file.lua

```diff
@@ -8,17 +8,20 @@ local find_file = require("nvim-tree.actions.finders.find-file").fn
 
 local M = {}
 
+---@param file string
 local function create_and_notify(file)
   events._dispatch_will_create_file(file)
   local ok, fd = pcall(vim.loop.fs_open, file, "w", 420)
   if not ok then
     notify.error("Couldn't create file " .. notify.render_path(file))
     return
   end
-  vim.loop.fs_close(fd)
+  vim.loop.fs_close(tonumber(fd) or error "This will never happen")
```

Comment:
Maybe it's not the best solution, but it works...

Reply:
This is a problem. We're struggling with similar issues at my workplace right now...

Reply:
Can we revert this one? It's correct and it passes checks.

Dealing with the `vim.` luals issues is a job for another day, if we even want to do it.